DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -18,7 +18,7 @@
   <!-- PAT Objects, and embedded data  -->
   <class name="pat::Electron"  ClassVersion="29">
    <version ClassVersion="29" checksum="1784986402"/>
-   <version ClassVersion="28" checksum="2518240031"/>
+   <version ClassVersion="28" checksum="1881133053"/>
```

Comment:
I think it was 27 (but version 28 was listed below anyway) and it told 
me to add 29.
I don't know why it changed the checksum for 28.

On 12.03.2015 13:22, Giovanni Petrucciani wrote:
>
> In DataFormats/PatCandidates/src/classes_def_objects.xml 
> <https://github.com/cms-sw/cmssw/pull/8221#discussion_r26296917>:
>
> > @@ -18,7 +18,7 @@
> >    <!-- PAT Objects, and embedded data  -->
> >    <class name="pat::Electron"  ClassVersion="29">
> >     <version ClassVersion="29" checksum="1784986402"/>
> > -   <version ClassVersion="28" checksum="2518240031"/>
> > +   <version ClassVersion="28" checksum="1881133053"/>
>
> Shouldn't you have added a |ClassVersion| 30 instead of modifying the 28?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/8221/files#r26296917>.
>

-- 
Matteo Sani
University of California, San Diego
+41227671577